airflow/api_fastapi/execution_api/datamodels/asset.py

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from airflow.api_fastapi.common.types import UtcDateTime
+from airflow.api_fastapi.core_api.base import BaseModel
+
+
+class DagScheduleAssetReference(BaseModel):
+    """DAG schedule reference schema for assets."""
+
+    dag_id: str
+    created_at: UtcDateTime
+    updated_at: UtcDateTime
+
+
+class TaskOutletAssetReference(BaseModel):
+    """Task outlet reference schema for assets."""
+
+    dag_id: str
+    task_id: str
+    created_at: UtcDateTime
+    updated_at: UtcDateTime
+
+
+class AssetResponse(BaseModel):
+    """Asset schema for responses."""
+
+    id: int
+    name: str
+    uri: str
+    group: str
+    extra: dict | None = None
+    created_at: UtcDateTime
+    updated_at: UtcDateTime
+    consuming_dags: list[DagScheduleAssetReference]
+    producing_tasks: list[TaskOutletAssetReference]
+    aliases: list[AssetAliasResponse]
+
+
+class AssetAliasResponse(BaseModel):
+    """Asset alias schema for responses."""
```

Comment:
These docs will likely appear in generated client -- so I don't think these doc strings make sense. Either we should make it make sense, or remove it and ignore the D*** lint rule here.

Reply:
True, will change